test/main.fmf

```diff
@@ -0,0 +1,2 @@
+duration: 20m
+test: common/run-tests
```

Comment:
I've fixed it to be able to run as it is. 
But you are right, this file could live also inside project root as I had it in the previous version, but it depends if we plan to use TMT `plans` right now. In case not, it is not so important to have it there, but if you wish I can move it there.

Reply:
@jscotka: That's fine, let's leave the files where they are for now, as long as that works for you.